forui/lib/src/widgets/date_picker/input/input_controller.dart

```diff
@@ -0,0 +1,215 @@
+import 'package:flutter/widgets.dart';
+
+import 'package:forui/forui.dart';
+import 'package:forui/src/widgets/date_picker/input/parser.dart';
+import 'package:intl/intl.dart';
+import 'package:meta/meta.dart';
+
+///
```

Comment:
Might be placed here by mistake.

```suggestion
```

Reply:
I's intended due to a bug with @internal not disabling the `public_api` lint.